### PR TITLE
Alert & button

### PR DESCRIPTION
**Pull request by anandkumarcoder**
Feat: Alert and button

image
image
image
image
image

### CODE REVIEW

File: Styles/alert.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  }
+
+
+  
+  /* - alert with icon starts */
+
+  .material-icons{
+    
+     vertical-align: middle;
+     padding: 0 0.3em;
```
Comment (TejasShekar) [inline Styles/alert.css]: Please use one particular unit.

**Review by TejasShekar — COMMENTED** [review]
1. You have a css files that have all predefined spacing and typography classes but you seem to have not used it anywhere.
2. Remove all spacings and unnecessary comments.
3. Also use one particular unit consistently. You have a mix of rems and ems.

File: Styles/alert.css (hunk 1 above)
Comment (anandkumarcoder) [reply Styles/alert.css]: above three changes are made in the new commit. 
Pls approve if things are looking fine.

**Review by TejasShekar — APPROVED** [review]

File: Styles/alert.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
+/* importing external css */
+@import "/Styles/variable.css";
+/* --------- */
+
+/* importing font*/
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&family=Roboto&display=swap");
```
Comment (darshandamre) [inline Styles/alert.css]: no need to import font here

File: Styles/navbar.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+     
+      color: #1b97f5;
+    
+  } */
+  
+
+ 
+  
+  .nav-bar {
+
```
Comment (darshandamre) [inline Styles/navbar.css]: use formatter

File: Styles/variable.css [hunk 4]
```diff
@@ -0,0 +1,18 @@
+:root {
+  --primary-background-color: #fffbeb;
+ --primary-font: "Open Sans", sans-serif;
+  --primary-bg-alert: #1b98f5;
+  --secondary-bg-alert: #a8b8b9ce;
+  --success-bg-alert: #10b981;;
```
Comment (darshandamre) [inline Styles/variable.css]: wrong syntax, two semi colons

File: index.html [hunk 5]
```diff
@@ -1,13 +1,77 @@
...
+            Use component Falcon component library to make your website look
+            beautiful , faster and responsive
+          </h5>
+
+          <div class="button-container">
+            <button>Explore<i class="material-icons btn-only">east</i></button>
+          </div>
+        </div>
+      </section>
+      <!-- <aside></aside> -->
```
Comment (darshandamre) [inline index.html]: remove if unnecessary

File: Styles/variable.css (hunk 4 above)
Comment (anandkumarcoder) [reply Styles/variable.css]: correction made

File: Styles/alert.css (hunk 2 above)
Comment (anandkumarcoder) [reply Styles/alert.css]: Correction made

File: index.html (hunk 5 above)
Comment (anandkumarcoder) [reply index.html]: change made

**Pull request merged**